treasury/TreasurySpending.md

```diff
@@ -0,0 +1,36 @@
+# Treasury Spending
+
+### ChaosDAO Treasury Goals and Philosophy:
+ChaosDAO's ideal use of CD treasury funding is to support the Polkadot and Kusama ecosystem and community.
+
+More specifically:
+* Support common good code/tool development. We don’t think we have enough funds to significantly contribute yet, so CD is still an unfunded knowledge sharing tech innovation hub currently. For instance, subscrape. Taylor would love a Wordpress plugin. Spazcoin would love better multisig DAO tooling.
+* Funding on-chain public identity for ChaosDAO as a generally impartial community organization (see Nomination Pools).
+* Operating community infrastructure such as collators. To start with this has primarily been a revenue source for CD, but once people have bandwidth in the future, we can also support unprofitable collators to encourage decentralization. 
+
+Some general guidelines to follow:
+
+* Continue to build and grow the Treasury so that we can take on bigger challenges in the future. Never let expenditures be larger than recent revenue. Expect Polkadot ecosystem tokens to be worth much more in the bull market where we can do more with what we have slowly earned over time.
+* Try to use the tokens within the ecosystem where they were earned (Astar, Moonriver).
+
+
+### Expenditures from the ChaosDAO treasury so far:
+* In Jan'22, DAOists approved sending half of our small MOVR treasury to the SuperColony "Ukranian Humanitarian Fund". About $1.6k (20 MOVR)
+* In Nov'22, we created DOT and KSM "Nomination Pools" to support independent community-focused valiators. Used ~2 KSM and 225 DOT (out of ~650 DOT earned by the Interlay vault). We think this will show important public community leadership.
```

Comment:
After validators I would remove the period and replace it with a comma, then swap out "used" for "using" (lower case U). Also need to put the closing parenthesis mark after the word "leadership" but before the period.

Reply:
corrected the first part of your comment (and `valiators` was misspelled.)

didn't understand why to put a parens after `leadership`. The parenthesis section was just to indicate how much of our DOT holdings were put into the Nominator Pool acct.